certif/app/routes/session-supervising.js

```diff
@@ -14,21 +14,28 @@ export default class SessionSupervisingRoute extends Route {
 
   afterModel(model) {
     this.poller = setInterval(async () => {
-      await this.store.queryRecord('session-for-supervising', { sessionId: model.id });
+      try {
+        await this.store.queryRecord('session-for-supervising', { sessionId: model.id });
+      } catch (_) {
+        this.#stopPolling();
+      }
```

Comment:
```suggestion
    this.poller = setInterval(
      () => this.store.queryRecord('session-for-supervising', { sessionId: model.id })
        .catch(this.#stopPolling)
    );
```

Ça me parait plus facile à lire.
Qu'en pensez-vous ?